Dockerfile

```diff
@@ -485,8 +485,10 @@ WORKDIR /
 USER root
 
 # Add buildscript for local testing
-ADD run-build-functions.sh /usr/local/bin/run-build-functions.sh
-ADD run-build.sh /usr/local/bin/build
+ENV BUILD_BIN_DIR "/opt/build-bin"
+RUN mkdir -p ${BUILD_BIN_DIR}
+ADD run-build-functions.sh ${BUILD_BIN_DIR}/run-build-functions.sh
+ADD run-build.sh ${BUILD_BIN_DIR}/build
```

Comment:
In my opinion, we're making too much use of environment variables to dry up scripts and Dockerfiles. While get rid of some repetition, we lose the ability to glance at a line and see exactly what's happening.

Given that we don't currently write unit tests for things like `run-build-functions.sh`, I think we should keep them more explicit and un-code-like.

Reply:
Sounds good to me - I was iffy on this anyway. I'll change it back to explicit paths.